handbook/engineering/hiring/engineering-manager-backend-platform.md

```diff
@@ -0,0 +1,68 @@
+# Engineering Manager - Backend Platform
+
+We are looking for an engineering manager to join the team that is building the backend platform upon which product engineering teams can iterate quickly and safely. The team's mission is to promote and champion consistency and simplicity across all backend code by building shared tools, libraries and infrastructure for common use cases, scale and maintain said infrastructure and ultimately create leverage for and accelerate other teams.
+
+The [Cloud team](https://about.sourcegraph.com/handbook/engineering/cloud) team will become the Backend Platform team once we hire a Product Manager to drive the Cloud project forward.
+
+## Responsibilities
+
+You will:
+
+- Facilitate a inclusive and cohesive team environment where everyone is happy, motivated, and productive.
+- Model, teach, and reinforce [our company values](../../company/values.md) and [our guiding engineering principles](index.md#guiding-principles) to the team.
+- Ensure the team has clear incremental goals that are documented and are always up-to-date.
+- Regularly communicate the team's progress toward their goals as well as changes in team goals to appropriate stakeholders.
+- Support and coach teammates to grow in their careers and fulfill their responsibilities.
+- Grow the team in a sustainable way so that the team can accomplish more over time.
+
+## Qualifications
+
+- You have 2+ years of experience directly managing at least 6 full-time software engineers. We expect you already know how to lead a team.
+- You have done some amount of coding in the past 2 years (for example: at work or on a personal project) to keep your skills up-to-date. We expect that you are able to review and provide valuable feedback on code, architecture, and designs.
+- You are able to clearly communicate your past accomplishments verbally (example: in an interview) and in writing (examples: in your resume, in your answers to our application questions).
+- You have experience breaking up complex technical problems into smaller achievable milestones that the team can iterate on.
+
+## Nice to haves
+
+- You have experience designing, architecting, scaling, and optimizing systems that span multiple layers of a tech stack (example: frontend, backend, CLI).
+- You have grown at least one junior engineer to a senior+ level.
+- You have designed a software engineer interview process from first principles.
+- You have built and delivered an application that works on multiple platforms (examples: different operating systems, cloud versus on-premise)
+
```

Comment:
Is this simply copy/paste from full stack role? Feels kind of generic for this particular role, so my proposal would be to delete these things by default and then add anything back that would be a specific nice-to-have for this particular role (e.g. past experience as manager or IC on team building a cloud product, past experience as manager or IC on team building backend application infrastructure)
```suggestion
```

Reply:
Yes, these were copied from the code intel manager job req. I'm fine leaving these out.